### PR TITLE
Added regression test for RF.orient() and improved docs of RF.dcm()

### PR DESCRIPTION
**Pull request by moorepants**
#### References to other Issues or PRs

This is a regression test for the bug introduced in PR #14758, identified in #16824, and solved by PR #16828.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by moorepants** [comment]
Here is a render of the new docstring:
Selection_594

**Comment by moorepants** [comment]
This is ready for review.

**Comment by moorepants** [comment]
@sympy/mechanics This is ready for review.

**Comment by moorepants** [comment]
I think that this is ready to go in. Can I get an approval from a reviewer?

**Comment by moorepants** [comment]
This has been open for a week with no review attention. I would like to merge it. Can someone please check it off. It is pretty minor, no changes to functionality. Just improved docs and an additional regression test.

File: sympy/physics/vector/frame.py [hunk 1]
```diff
@@ -377,32 +377,74 @@ def ang_vel_in(self, otherframe):
...
+
+        .. math::
+
+           \begin{bmatrix}
+             \hat{\mathbf{b}}_1 \\
+             \hat{\mathbf{b}}_2 \\
+             \hat{\mathbf{b}}_3
+           \end{bmatrix}
+           =
+           ^A\mathbf{R}^B
```
Comment (asmeurer) [inline sympy/physics/vector/frame.py]: You should check that this renders correctly. I thought you had to do `{}^A` in LaTeX but I could be mistaken.

Comment (moorepants) [reply sympy/physics/vector/frame.py]: Yes, I think that is the trick. I couldn't remember it.

Comment (moorepants) [reply sympy/physics/vector/frame.py]: I wrote it probably a hundred times in my dissertation...but forgot.

**Comment by moorepants** [comment]
Ok, I fixed the left superscripts. Ready to go in from my end.

**Comment by moorepants** [comment]
There are three seemingly very unrelated build failures.

**Comment by oscarbenjamin** [comment]
I've restarted the tests because they previously failed due to a problem with tensorflow that is now fixed.

**Comment by moorepants** [comment]
Can this be merged?

**Comment by oscarbenjamin** [comment]
> Can this be merged?

Yea :)

**Pull request merged**